### PR TITLE
Added a script to list the last runs of all cgal's workflows

### PR DESCRIPTION
**Pull request by SaillantNicolas**
## Summary of Changes

Addition of a script to list all the last runs of cgal workflows.

### CODE REVIEW

File: Scripts/developer_scripts/list_cgal_workflows_last_run.sh [hunk 1]
```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+echo "  workflow  |  runs on  |  status of last run  |  date  |  file  |"
```
Comment (lrineau) [inline Scripts/developer_scripts/list_cgal_workflows_last_run.sh:2]: Check the Markdown syntax:

It should be:
```suggestion
echo "| workflow  |  runs on  |  status of last run  |  date  |  file  |"
echo "| --------- | --------- | -------------------- | ------ | ------ |"
```

And remove the `echo "----------"` between lines.

Comment (lrineau) [reply Scripts/developer_scripts/list_cgal_workflows_last_run.sh:2]: See https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/organizing-information-with-tables#creating-a-table

**Comment by lrineau** [comment]
The current version of the script gives that result:

|  workflow  |  runs on  |  status of last run  |  date  |  file  |
| ---------- | --------- | -------------------- | ------ | ------ |
| "Tests" | "schedule" | "completed" - "failure" | "2022-11-07T08:59:03Z"| CGAL/cgal-testsuite-dockerfiles/.github/workflows/tests.yml
| "remove_labels" | "pull_request_target" | "completed" - "skipped" | "2022-11-09T14:17:05Z"| CGAL/cgal/.github/workflows/Remove_labels.yml
| "Documentation" | "issue_comment" | "queued" - null | "2022-11-09T14:36:51Z"| CGAL/cgal/.github/workflows/build_doc.yml
| "CMake Test Merge Branch" | "pull_request" | "in_progress" - null | "2022-11-09T14:17:05Z"| CGAL/cgal/.github/workflows/checks.yml
| "CMake Testsuite" | "pull_request" | "completed" - "success" | "2022-11-09T14:17:05Z"| CGAL/cgal/.github/workflows/cmake-all.yml
| "Documentation Removal" | "pull_request_target" | "completed" - "success" | "2022-11-08T13:04:39Z"| CGAL/cgal/.github/workflows/delete_doc.yml
| "Test Polyhedron Demo" | "pull_request" | "in_progress" - null | "2022-11-09T14:17:05Z"| CGAL/cgal/.github/workflows/demo.yml
| "Filter Testsuite" | "issue_comment" | "completed" - "skipped" | "2022-11-09T14:36:51Z"| CGAL/cgal/.github/workflows/filter_testsuite.yml
| "CMake Build for Doxygen" | "push" | "completed" - "failure" | "2021-07-29T15:03:02Z"| CGAL/doxygen/.github/workflows/build_cmake.yml
| "CI Tests" | "pull_request" | "completed" - "failure" | "2022-11-03T10:56:42Z"| CGAL/cgal-swig-bindings/.github/workflows/tests.yml

**Comment by lrineau** [comment]
I have fixed a typo. See 280a8c6c8b5e62700c833101012b5859e42716a8.

Now we have:

|  workflow  | branch | event | runs on  |  status of last run  |  date  |  file  |
| ---------- | ------ | ----- | -------- | -------------------- | ------ | ------ |
| "Tests" | master | push:<br/>pull_request:<br/>workflow_dispatch:<br/>schedule:<br/> | "schedule" | "completed" - "success" | "2022-11-20T01:35:57Z"| CGAL/cgal-testsuite-dockerfiles/.github/workflows/tests.yml
| "remove_labels" | gsoc2022-isosurface | pull_request_target:<br/> | "pull_request_target" | "completed" - "skipped" | "2022-11-21T07:29:41Z"| CGAL/cgal/.github/workflows/Remove_labels.yml
| "Documentation" | master | issue_comment:<br/> | "issue_comment" | "completed" - "success" | "2022-11-21T08:29:41Z"| CGAL/cgal/.github/workflows/build_doc.yml
| "CMake Test Merge Branch" | gsoc2022-isosurface | [push,<br/>pull_request]<br/> | "pull_request" | "completed" - "failure" | "2022-11-21T07:31:24Z"| CGAL/cgal/.github/workflows/checks.yml
| "CMake Testsuite" | gsoc2022-isosurface | [push,<br/>pull_request]<br/> | "pull_request" | "completed" - "success" | "2022-11-21T07:31:24Z"| CGAL/cgal/.github/workflows/cmake-all.yml
| "Documentation Removal" | Polyhedron_demo-mesh_3_plugin_cleaning-jtournois | pull_request_target:<br/> | "pull_request_target" | "completed" - "success" | "2022-11-15T16:02:23Z"| CGAL/cgal/.github/workflows/delete_doc.yml
| "Test Polyhedron Demo" | gsoc2022-isosurface | [push,<br/>pull_request]<br/> | "pull_request" | "completed" - "success" | "2022-11-21T07:31:24Z"| CGAL/cgal/.github/workflows/demo.yml
| "Filter Testsuite" | master | issue_comment:<br/> | "issue_comment" | "completed" - "skipped" | "2022-11-21T08:29:41Z"| CGAL/cgal/.github/workflows/filter_testsuite.yml
| "CMake Build for Doxygen" | release_1_9_1_patched | null<br/> | "push" | "completed" - "failure" | "2021-07-29T15:03:02Z"| CGAL/doxygen/.github/workflows/build_cmake.yml
| "CI Tests" | add-python-3.11 | push:<br/>pull_request:<br/> | "pull_request" | "completed" - "failure" | "2022-11-03T10:56:42Z"| CGAL/cgal-swig-bindings/.github/workflows/tests.yml
| "CI" | master | pull_request:<br/>push:<br/>schedule:<br/> | "schedule" | "completed" - "success" | "2022-11-20T01:28:35Z"| CGAL/cgal-paraview-plugins/.github/workflows/ci.yml
| "Receive_PR" | master | [push,<br/>pull_request]<br/> | "push" | "completed" - "success" | "2022-11-15T12:43:08Z"| CGAL/cgal-web/.github/workflows/Receive_PR.yml
| "Deploy Jekyll with GitHub Pages dependencies preinstalled" | master | workflow_run:<br/> | "workflow_run" | "completed" - "success" | "2022-11-15T12:44:50Z"| CGAL/cgal-web/.github/workflows/jekyll-pages.yml
| "pages build and deployment" | gh-pages | null<br/> | "dynamic" | "completed" - "success" | "2022-11-04T14:26:26Z"| CGAL/cgal-web/dynamic/pages/pages-build-deployment
| "ci" | main | push:<br/>workflow_dispatch:<br/> | "push" | "completed" - "success" | "2022-10-19T08:14:40Z"| CGAL/cgal-python-wheel/.github/workflows/docker-publish.yml

**Comment by lrineau** [comment]
| repo |  workflow  | branch | event | runs on  |  status of last run  | annotation |  date  |  file  |
| :--: | :--------: | :----: | :---: | :------: | :------------------: | :--------: | :----: | :----: |
| CGAL/cgal-testsuite-dockerfiles | "Tests" | master | push:<br/>pull_request:<br/>workflow_dispatch:<br/>schedule:<br/> | "schedule" | "completed" - "success" | ***-*** | "2022-11-27T01:34:32Z" | CGAL/cgal-testsuite-dockerfiles/.github/workflows/tests.yml
| CGAL/cgal | "remove_labels" | P3M3-Undocumented_polyhedral_domain_example-GF | pull_request_target:<br/> | "pull_request_target" | "completed" - "skipped" | ***-*** | "2022-11-25T15:37:44Z" | CGAL/cgal/.github/workflows/Remove_labels.yml
| CGAL/cgal | "Documentation" | master | issue_comment:<br/> | "issue_comment" | "completed" - "success" | ***-*** | "2022-11-28T07:34:31Z" | CGAL/cgal/.github/workflows/build_doc.yml
| CGAL/cgal | "CMake Test Merge Branch" | Add_Script_To_list_workflows-runs | [push,<br/>pull_request]<br/> | "pull_request" | "completed" - "success" | ***-*** | "2022-11-25T14:15:59Z" | CGAL/cgal/.github/workflows/checks.yml
| CGAL/cgal | "CMake Testsuite" | Add_Script_To_list_workflows-runs | [push,<br/>pull_request]<br/> | "pull_request" | "completed" - "success" | ***-*** | "2022-11-25T14:15:59Z" | CGAL/cgal/.github/workflows/cmake-all.yml
| CGAL/cgal | "Documentation Removal" | Aos_2-fixes-efif | pull_request_target:<br/> | "pull_request_target" | "completed" - "success" | ***-*** | "2022-11-25T09:16:38Z" | CGAL/cgal/.github/workflows/delete_doc.yml
| CGAL/cgal | "Test Polyhedron Demo" | Add_Script_To_list_workflows-runs | [push,<br/>pull_request]<br/> | "pull_request" | "completed" - "success" | ***-*** | "2022-11-25T14:15:59Z" | CGAL/cgal/.github/workflows/demo.yml
| CGAL/cgal | "Filter Testsuite" | master | issue_comment:<br/> | "issue_comment" | "completed" - "skipped" | ***-*** | "2022-11-28T07:34:31Z" | CGAL/cgal/.github/workflows/filter_testsuite.yml
| CGAL/doxygen | "CMake Build for Doxygen" | release_1_9_1_patched | null<br/> | "push" | "completed" - "failure" | ***failure*** | "2021-07-29T15:03:02Z" | CGAL/doxygen/.github/workflows/build_cmake.yml
| CGAL/cgal-swig-bindings | "CI Tests" | add-python-3.11 | push:<br/>pull_request:<br/> | "pull_request" | "completed" - "failure" | ***-*** | "2022-11-03T10:56:42Z" | CGAL/cgal-swig-bindings/.github/workflows/tests.yml
| CGAL/cgal-paraview-plugins | "CI" | master | pull_request:<br/>push:<br/>schedule:<br/> | "schedule" | "completed" - "success" | ***warning*** | "2022-11-27T01:28:12Z" | CGAL/cgal-paraview-plugins/.github/workflows/ci.yml
| CGAL/cgal-web | "Receive_PR" | master | [push,<br/>pull_request]<br/> | "push" | "completed" - "success" | ***-*** | "2022-11-15T12:43:08Z" | CGAL/cgal-web/.github/workflows/Receive_PR.yml
| CGAL/cgal-web | "Deploy Jekyll with GitHub Pages dependencies preinstalled" | master | workflow_run:<br/> | "workflow_run" | "completed" - "success" | ***-*** | "2022-11-15T12:44:50Z" | CGAL/cgal-web/.github/workflows/jekyll-pages.yml
| CGAL/cgal-web | "pages build and deployment" | gh-pages | null<br/> | "dynamic" | "completed" - "success" | ***-*** | "2022-11-04T14:26:26Z" | CGAL/cgal-web/dynamic/pages/pages-build-deployment
| CGAL/cgal-python-wheel | "ci" | main | push:<br/>workflow_dispatch:<br/> | "push" | "completed" - "success" | ***-*** | "2022-10-19T08:14:40Z" | CGAL/cgal-python-wheel/.github/workflows/docker-publish.yml

| repo | dependabot |
| :--: | :--------: |
| CGAL/cgal-testsuite-dockerfiles | yes |
| CGAL/cgal | yes |
| CGAL/doxygen | no |
| CGAL/cgal-swig-bindings | yes |
| CGAL/cgal-paraview-plugins | no |
| CGAL/cgal-web | no |
| CGAL/cgal-python-wheel | yes |

**Comment by SaillantNicolas** [comment]
Now we have : 

| repo |  workflow  | branch | event | runs on  |  status of last run  | state | annotation |  date  | date since last runs |  file  |
| :--: | :--------: | :----: | :---: | :------: | :------------------: | :---: | :--------: | :----: | :------------------: | :----: |
| CGAL/cgal-testsuite-dockerfiles | "Tests" | master | push:<br/>pull_request:<br/>workflow_dispatch:<br/>schedule:<br/> | schedule | completed - success | "active" | ***-*** | 2023-01-15T01:33:02Z | 3 days | CGAL/cgal-testsuite-dockerfiles/.github/workflows/tests.yml
| CGAL/cgal | "remove_labels" | Add_Script_To_list_workflows-runs | pull_request_target:<br/> | pull_request_target | completed - skipped | "active" | ***-*** | 2023-01-18T17:57:10Z | 0 days | CGAL/cgal/.github/workflows/Remove_labels.yml
| CGAL/cgal | "Documentation" | master | issue_comment:<br/> | issue_comment | completed - failure | "active" | ***failure*** | 2023-01-18T16:43:57Z | 0 days | CGAL/cgal/.github/workflows/build_doc.yml
| CGAL/cgal | "CMake Test Merge Branch" | Add_Script_To_list_workflows-runs | [push,<br/>pull_request]<br/> | pull_request | in_progress - null | "active" | ***-*** | 2023-01-18T17:57:13Z | 0 days | CGAL/cgal/.github/workflows/checks.yml
| CGAL/cgal | "CMake Testsuite" | Add_Script_To_list_workflows-runs | [push,<br/>pull_request]<br/> | pull_request | in_progress - null | "active" | ***-*** | 2023-01-18T17:57:12Z | 0 days | CGAL/cgal/.github/workflows/cmake-all.yml
| CGAL/cgal | "Documentation Removal" | feature/bug_documentation_spell_20221113 | pull_request_target:<br/> | pull_request_target | completed - success | "active" | ***-*** | 2023-01-12T18:04:49Z | 5 days | CGAL/cgal/.github/workflows/delete_doc.yml
| CGAL/cgal | "Test Polyhedron Demo" | Add_Script_To_list_workflows-runs | [push,<br/>pull_request]<br/> | pull_request | in_progress - null | "active" | ***-*** | 2023-01-18T17:57:12Z | 0 days | CGAL/cgal/.github/workflows/demo.yml
| CGAL/cgal | "Filter Testsuite" | master | issue_comment:<br/> | issue_comment | completed - skipped | "active" | ***-*** | 2023-01-18T16:43:57Z | 0 days | CGAL/cgal/.github/workflows/filter_testsuite.yml
| CGAL/doxygen | "CMake Build for Doxygen" | release_1_9_1_patched | null<br/> | push | completed - failure | "active" | ***failure*** | 2021-07-29T15:03:02Z | 538 days | CGAL/doxygen/.github/workflows/build_cmake.yml
| CGAL/cgal-swig-bindings | "CI Tests" | main | push:<br/>pull_request:<br/>workflow_dispatch:<br/> | workflow_dispatch | completed - failure | "active" | ***failure*** | 2023-01-03T09:06:04Z | 15 days | CGAL/cgal-swig-bindings/.github/workflows/tests.yml
| CGAL/cgal-paraview-plugins | "CI" | master | pull_request:<br/>push:<br/>schedule:<br/> | schedule | completed - success | "disabled_inactivity" | ***warning*** | 2022-11-27T01:28:12Z | 52 days | CGAL/cgal-paraview-plugins/.github/workflows/ci.yml
| CGAL/cgal-web | "Receive_PR" | Add_workflow_dispatch | [push,<br/>pull_request]<br/> | pull_request | completed - success | "active" | ***-*** | 2023-01-18T11:04:03Z | 0 days | CGAL/cgal-web/.github/workflows/Receive_PR.yml
| CGAL/cgal-web | "Deploy Jekyll with GitHub Pages dependencies preinstalled" | master | workflow_run:<br/> | workflow_run | completed - failure | "active" | ***-*** | 2023-01-18T11:05:24Z | 0 days | CGAL/cgal-web/.github/workflows/jekyll-pages.yml
| CGAL/cgal-web | "pages build and deployment" | gh-pages | null<br/> | dynamic | completed - success | "active" | ***-*** | 2022-11-04T14:26:26Z | 75 days | CGAL/cgal-web/dynamic/pages/pages-build-deployment
| CGAL/cgal-python-wheel | "ci" | main | push:<br/>workflow_dispatch:<br/> | workflow_dispatch | completed - success | "active" | ***-*** | 2022-11-28T08:45:18Z | 51 days | CGAL/cgal-python-wheel/.github/workflows/docker-publish.yml

| repo | dependabot |
| :--: | :--------: |
| CGAL/cgal-testsuite-dockerfiles | yes |
| CGAL/cgal | yes |
| CGAL/doxygen | no |
| CGAL/cgal-swig-bindings | yes |
| CGAL/cgal-paraview-plugins | no |
| CGAL/cgal-web | no |
| CGAL/cgal-python-wheel | yes |

**Pull request merged**